numba_dppy/spirv_generator.py

```diff
@@ -77,17 +77,43 @@ def optimize(self, ipath, opath):
             ]
         )
 
-    def generate(self, ipath, opath):
+    def generate(self, ipath, opath, llvm_spirv_args):
         # DRD : Temporary hack to get SPIR-V code generation to work.
         # The opt step is needed for:
         #     a) generate a bitcode file from the text IR file
         #     b) hoist all allocas to the enty block of the module
-
         # Get optimization level from NUMBA_OPT
         opt_level_option = f"-O{config.OPT}"
 
         check_call(["opt", opt_level_option, "-o", ipath + ".bc", ipath])
-        check_call(["llvm-spirv", "-o", opath, ipath + ".bc"])
+
+        if dppy_config.NATIVE_FP_ATOMICS == 1:
+            llvm_spirv_root = dppy_config.LLVM_SPIRV_ROOT
+
+            if llvm_spirv_root == "":
+                # try to find ONEAPI root
+                possible_oneapi_roots = ["/opt/intel/oneapi", "$A21_SDK_ROOT"]
```

Comment:
We cannot have these in our code-base. Users should set it themselves using envars.